pkg/clusterversion/cockroach_versions.go

```diff
@@ -502,10 +503,14 @@ var versionsSingleton = keyedVersions([]keyedVersion{
 	},
 	{
 		// VersionAlterSystemJobsTable is a version which modified system.jobs table
-		//
 		Key:     VersionAlterSystemJobsAddCreatedByColumns,
 		Version: roachpb.Version{Major: 20, Minor: 1, Unstable: 6},
 	},
+	{
+		// VersionAddScheduledJobsTable is a version which adds system.scheduled_jobs table
```

Comment:
(nit: also while we're here, could we add a `.` to the end of this comment and the one above)